src/index.ts

```diff
@@ -2,21 +2,18 @@ import { SignedWrappedDocument, v2, v3, WrappedDocument } from "@govtechsg/open-
 import { verificationBuilder } from "./verifiers/verificationBuilder";
 import { Verifier, Verifiers } from "./types/core";
 import { openAttestationHash } from "./verifiers/documentIntegrity/hash/openAttestationHash";
-import { Identity, openAttestationDnsTxt } from "./verifiers/issuerIdentity/dnsText/openAttestationDnsTxt";
 import { isValid } from "./validator";
 import { openAttestationEthereumTokenRegistryStatus } from "./verifiers/documentStatus/tokenRegistryStatus/openAttestationEthereumTokenRegistryStatus";
 import { openAttestationEthereumDocumentStoreStatus } from "./verifiers/documentStatus/documentStoreStatus/openAttestationEthereumDocumentStoreStatus";
 import { OpenAttestationDidSignedDocumentStatus } from "./verifiers/documentStatus/didSignedDocumentStatus";
-import { OpenAttestationDidSignedDidIdentityProof } from "./verifiers/issuerIdentity/didIdentityProof";
-import { OpenAttestationDnsDid } from "./verifiers/issuerIdentity/dnsDidProof";
+import { OpenAttestationIssuerIdentityVerifier } from "./verifiers/issuerIdentity";
 
 const openAttestationVerifiers: Verifiers[] = [
   openAttestationHash,
   openAttestationEthereumTokenRegistryStatus,
   openAttestationEthereumDocumentStoreStatus,
-  openAttestationDnsTxt,
-  OpenAttestationDnsDid,
   OpenAttestationDidSignedDocumentStatus,
+  OpenAttestationIssuerIdentityVerifier,
 ];
 
 const verify = verificationBuilder<
```

Comment:
This becomes harder to reuse:

```
const x = verifierX; // new identity verifier)

const verify = verificationBuilder([openAttestationHash,
  openAttestationEthereumTokenRegistryStatus,
  openAttestationEthereumDocumentStoreStatus,
  OpenAttestationDidSignedDocumentStatus
  issuerIdentityVerifierBuilder([...defaultIssuerIdentityProofVerifiers, verifierX])])
```

Reply:
Any suggestions?
I think we need to separate ease of reuse and correctness. Also, I think a sensible default will help with the ease of reuse.